cql3/query_processor.hh

```diff
@@ -341,9 +347,13 @@ public:
     // note: optimized for convenience, not performance.
     future<::shared_ptr<untyped_result_set>> execute_internal(
             const sstring& query_string,
-            db::consistency_level,
-            const std::initializer_list<data_value>&,
-            cache_internal cache);
+            db::consistency_level cl,
+            const std::initializer_list<data_value>& values,
+            cache_internal cache) {
+        auto qs = query_state_for_internal_call();
+        co_return co_await execute_internal(query_string, cl, qs, values, cache);
+    }
+
     future<::shared_ptr<untyped_result_set>> execute_internal(
```

Comment:
I think this patch (cql3: query_processor: define execute_internal overload in header) is no longer needed.

Reply:
right